cwt/cose_message.py

```diff
@@ -334,5 +337,6 @@ def attach_payload(self: Self, payload: bytes) -> COSEMessage:
 
         if self._payload is not None:
```

Comment:
For now, if you want to implement holding the detached payload in a COSEMessage structure:

```suggestion
        if self._msg[2] is not None:
```